build_package.py

```diff
@@ -1855,6 +1862,15 @@ class BadChangeLogFormat(Exception):
         help="Enable coverage analysis. Can be used in smoketests. Only works with docker/k8s.",
     )
 
+    parser.add_option(
+        "",
+        "--only-container-tarball",
+        dest="only_container_tarball",
+        action="store_true",
+        default=False,
+        help="Only works with docker/k8s.",
```

Comment:
Please clarify this help string since it's not really clear from the current help string what's the purpose of this option and what purpose it serves.